source/PlayerInfo.cpp

```diff
@@ -551,26 +551,34 @@ void PlayerInfo::IncrementDate()
 				, Messages::Importance::Highest);
 	
 	// Check what salaries and tribute the player receives.
-	int64_t total[2] = {0, 0};
-	static const string prefix[2] = {"salary: ", "tribute: "};
-	for(int i = 0; i < 2; ++i)
-	{
-		auto it = conditions.lower_bound(prefix[i]);
-		for( ; it != conditions.end() && !it->first.compare(0, prefix[i].length(), prefix[i]); ++it)
-			total[i] += it->second;
-	}
-	if(total[0] || total[1])
+	auto getIncome = [&](string prefix) {
```

Comment:
```suggestion
	auto GetIncome = [&](string prefix) {
```

Lambdas are functions and as such they get to be capitalized like them :)